client/login/controller.js

```diff
@@ -14,8 +15,31 @@ const enhanceContextWithLogin = ( context ) => {
 		params: { flow, isJetpack, isGutenboarding, socialService, twoFactorAuthType },
 		path,
 		query,
+		isServerSide,
 	} = context;
 
+	// Process a social login handoff from /start/user.
+	if ( query?.email_address && query?.service && query?.access_token && query?.id_token ) {
```

Comment:
I'm not sure if this is a fair concern, but are there any security concerns putting the access token in the URL?

Reply:
> I'm not sure if this is a fair concern, but are there any security concerns putting the access token in the URL?

@noahtallen It's a great concern!

The access token and id token are: 1) passed along in another URL, and 2) stored in local storage. As such, I think it's fine to expose them again in a URL, although I agree it's less than ideal.

Reply:
Cool. That makes sense to me 👍